core/generate_sales_output.py

```diff
@@ -38,25 +38,45 @@ def generate_sales_output(
     # Read from logs file
     logs_df = pd.read_csv(logs_file)
 
-    # Filter for OpenSea v1 and v2 marketplace contracts
+    # Filter for marketplace contracts
     logs_df = logs_df.loc[
         logs_df["address"].isin(
             [
-                "0x7be8076f4ea4a4ad08075c2508e481d6c946d12b",
-                "0x7f268357a8c2552623316e2562d90e642bb538e5",
+                "0x7be8076f4ea4a4ad08075c2508e481d6c946d12b",  # OpenSea v1
+                "0x7f268357a8c2552623316e2562d90e642bb538e5",  # OpenSea v2
+                "0x59728544b08ab483533076417fbbb2fd0b17ce3a",  # LooksRare
             ]
         )
     ]
 
-    # Filter for topic0 equal to the kecakk-256 hash of OrdersMatched(bytes32,bytes32,address,address,uint256,bytes32)
-    event_signature_hash = (
-        "0xc4109843e0b7d514e4c093114b863f8e7d8d9a458c372cd51bfe526b588006c9"
-    )
-    logs_df = logs_df.loc[logs_df["topics"].str[:66] == event_signature_hash]
+    # Filter for topic0 equal to:
```

Comment:
Would be nice to continue to say it's the keccak hash of those function signatures.